packages/at_client/lib/src/client/at_client_impl.dart

```diff
@@ -954,4 +973,27 @@ class AtClientImpl implements AtClient {
     _preference = preference;
     _namespace = namespace;
   }
-}
+
+  Future<AtChops> _createAtChopsInstance() async {
```

Comment:
Don't think we need this method any more?

Reply:
yeah.i will remove this.